### DISCUSSION TITLE
README에 어떤 내용이 들어가면 좋을까요

### DISCUSSION THREAD

**Discussion by eastshine-high** [CodeSoom/DDD-Kurly-Clone-Promotion#29] [General]
의견을 남겨주세요!

**Comment by eastshine-high** [comment]
저는 간단한 프로젝트 설명이 있으면 괜찮을 것 같습니다. 또 도메인 주도 설계 스터디이니까 유비쿼터스 언어 정의를 여기에 넣어보는 게 어떨까 생각해 봤습니다.

**Comment by hannut91** [comment]
* 프로젝트 소개
* 기능 소개
* 사용법 => URL
* API 문서 안내
* 위키 안내
* 설치 방법
* 실행 방법

**Comment by etff** [comment]
기술스택이나 아키텍쳐 분석 설계를 두는 것도 좋을 것같네요.
DDD 스터디이니깐 코드 예제로 유비쿼터스 언어가 반영된 것들을 올려보는 것도 좋고요.
